<launch>
    <node name="map_server" pkg="map_server" type="map_server" args="$(find stage_simulation)/config/map_8thfloor.yaml">
        <param name="frame_id" value="/map" />
    </node>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find stage_simulation)/config/simulation.rviz"/>    

    <include file="$(find stage_simulation)/launch/stage.launch"/>
    <include file="$(find stage_simulation)/launch/amcl.launch" />
    
    <node pkg="active_perception_controller" type="person_estimator" name="estimator">
	<param name="step_duration" type="double" value="0.2" />
	<param name="num_particles" type="int" value="5000" />
	<param name="sigma_person" type="double" value="0.05" />
	<param name="rfid_map_resolution" type="double" value="0.06" />
	<param name="rfid_prob_map" type="string" value="$(find active_perception_controller)/config/sensormodel.png" />

    </node>

    <!--node pkg="active_perception_controller" type="motion_planner.py" name="entropy_motion_planner" output="screen"/-->
    
    <!--node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
        <rosparam file="$(find stage_simulation)/config/costmap_common_params_stage.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find stage_simulation)/config/costmap_common_params_stage.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find stage_simulation)/config/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find stage_simulation)/config/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find stage_simulation)/config/base_local_planner_params.yaml" command="load" />
        <param name="base_global_planner" type="string" value="navfn/NavfnROS" />
        <param name="controller_frequency" type="double" value="20.0" />
        <rosparam param="recovery_behaviors">[{name: conservative_reset, type: clear_costmap_recovery/ClearCostmapRecovery},
                                              {name: aggressive_reset, type: clear_costmap_recovery/ClearCostmapRecovery}]</rosparam>
        <param name="planner_frequency" type="double" value="0.1"/>
        <param name="planner_patience" type="double" value="2.0"/>
        <param name="controller_patience" type="double" value="60.0"/>
        <remap from="map" to="/map"/>
    </node-->
</launch>
